frontend/src/components/Collections/components/DeleteCollection/index.tsx

```diff
@@ -1,22 +1,28 @@
 import { Button, H6, Intent } from "@blueprintjs/core";
 import { IconNames } from "@blueprintjs/icons";
-import { navigate } from "@reach/router";
+import loadable from "@loadable/component";
+import { useNavigate } from "@reach/router";
 import React, { FC, useState } from "react";
 import { ROUTES } from "src/common/constants/routes";
 import { Collection } from "src/common/entities";
 import { useDeleteCollection } from "src/common/queries/collections";
-import Alert from "src/components/Alert";
 
+const AsyncAlert = loadable(
```

Comment:
Code split `src/components/Alert` out, so we only preload the component when the button is hovered or load the component when Alert is needed!